geode-connectors/src/test/java/org/apache/geode/connectors/jdbc/internal/configuration/ConnectorServiceTest.java

```diff
@@ -43,28 +43,6 @@ public void setUp() throws Exception {
     jaxbService.validateWith(local_xsd);
   }
 
-  @Test
-  public void connectorServiceCanBeCorrectlyMarshalled() {
```

Comment:
Looks like we still have connector-service in our jdbc xml (but we deleted the namespace). Should we keep this test and just remove the lines for connection or should jdbc xml be deleted?

Reply:
the connector-service will go away in another story related to the nested region-mapping

Reply:
we still have connector-service as explained. But this particular test was only testing the nested "connection" element that was removed. In this same class we kept the test for the nested region-mapping.